src/lib/config_parser.h

```diff
@@ -31,6 +31,16 @@ typedef bool (*parse_config_fn_t)(char *section, char *key, char *value, void *d
  */
 bool config_parse(const char *filename, parse_config_fn_t parse_config_fn, void *data);
 
+/**
+ * @brief Searches the INI style configuration file for the requested key within the
+ * specified section
+ * @param filename: the full path to the configuration file where the key will be searched
+ * @param section: the section from the config file we are interested in
+ * @param key: the key we want to get the value from
+ * @returns The value that matches the section and key
+ */
+char *config_get_value(const char *filename, const char *section, const char *key);
```

Comment:
@castulo If you need to query and get specific, may you can query the struct I create in 3rd-party_config.c. We can create a non-static method in 3rd-party_config.c file which queries a  a struct instead of a ini

Let me know what you think

Reply:
https://github.com/clearlinux/swupd-client/pull/1199/files#diff-80aeb8f47ce71855e9a9de63b5be69c6

I will have to change repo_config_deinit & repo_config_init from static to global so that you can call them at the start of 3rd-party each command. You can add the get_value function here and returns   values from the struct. Just a suggestion